std_msgs/Header header
string group_name
string eef_group_name
string box_name
geometry_msgs/Pose box_pose
geometry_msgs/Point box_size
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status